docs/connectors/mysql.asciidoc

```diff
@@ -958,6 +958,12 @@ Other than MySQL's `TIMESTAMP` data type, the other MySQL temporal types depend
 As of Debezium 0.7 `adaptive_time_microseconds` mode was introduced and is the default `time.precision.mode` for the MySQL connector. Mode `adaptive` was marked as deprecated.
 ====
 
+[NOTE]
+====
+As of Debezium 0.8 all time related types that does not have associated timezone are related to the timezone in which the database is operating.
```

Comment:
"Are related" sounds a bit vague. How about giving an example, clarifying that such columns are converted to UTC in the CDC messages?